doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mysql_sink.md

```diff
@@ -309,6 +324,14 @@ As any other NGSI-like sink, `NGSIMySQLSink` extends the base `NGSISink`. The me
 
 A `Batch` contains a set of `NGSIEvent` objects, which are the result of parsing the notified context data events. Data within the batch is classified by destination, and in the end, a destination specifies the MySQL table where the data is going to be persisted. Thus, each destination is iterated in order to compose a per-destination data string to be persisted thanks to any `MySQLBackend` implementation.
 
+    void capRecords(NGSIBatch batch, long maxRecords) throws EventDeliveryException;
+    
+This method is always called immediatelly after `persistBacth()`. The same destination tables that were upserted are now checked in terms of number of records: if the configured maximum (`persistence_policy.max_records`) is overcome for any of the updated tables, then as many oldest records are deleted as required until the maximum number of records is reached.
+    
+    void expirateRecords(long expirationTime);
+    
+This method is called in a periodical way (based on `persistence_policy.checking_time`), and if the configured expiration time (`persistence_policy.expiration_time`) is overcome for any of the records within any of the tables, then it is deleted.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1366/commits/536811ecc9e7c7382eac81509a89ad3a34121fdf